### ISSUE TITLE
Support keyboard navigation and improve screen reader accessibility in QuickFolders settings

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#555]
At the moment, QuickFolders settings is hard to navigate for users who cannot use mouse or have eye sight accessibility issues. Let's improve this!

For instance: 

the tabs that repesent the various pages of settings cannot be keyboard navigated, which makes their access very difficult to visually impaired people.

**Comment by RealRaven2000** [comment]
Here is a preview version that tries to make as much as possible accessible. there is still a gap with the palette pickers.

[QuickFolders-mx-6.10pre193.zip](https://github.com/user-attachments/files/19237683/QuickFolders-mx-6.10pre193.zip)

---

To try out this version, download the zip file and then drag it into Thunderbird Add-ons Manager (without extracting)
for users of screen reader you can also install this via the tools button after the "Find more addons" search text. activate the button with Enter then chose the 3rd menu entry "install addon from file...". Then select the zip file to install.
